eng/Packages.Data.props

```diff
@@ -11,7 +11,7 @@
     Dependency versions for Track 1 libraries.
   -->
   <ItemGroup Condition="'$(IsClientLibrary)' != 'true'">
-    <PackageReference Update="Microsoft.Azure.Amqp" Version="2.4.9" />
+    <PackageReference Update="Microsoft.Azure.Amqp" Version="2.4.11" />
```

Comment:
Do we actually want separate listings for Track 1 and Track 2 here?
/cc @jsquire @pakrym

Reply:
I'd prefer that we keep a single entry unless there's some incompatibility that impacts one library but not the other.

Reply:
I would recommend just removing this one and relying on the other entry since the dependency is allowed for Track 2 and Track 1.

Reply:
Actually.. the other entry is within the Track 2 block so I don't think it will work.

Reply:
@pakrym what do you think about a separate section that is not conditional for dependencies that are common for T1 and T2?

Reply:
I would prefer not to have it. T1 dependencies should only be updated when we specifically want to target T1.